### PR TITLE
Cobbduceus: (Baby) C2 Additions (Rhig,Troph) and Changes (Helbital)

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
It's the year of the rat sisters, and now it's time to cut out those pesky C3s in your life!

## About The Pull Request

Old Chem | New Chem | Changes
-- | -- | --
Helbital | N/A | Converts Helbital into an   easier to understand Chem. Is based entirely around stat   (crit/softcrit/notcrit) instead of crazy math, and heals brute based off all   4 'base' damagetypes. Not being in hardcrit gives you O2 damage and doesn't   let you roll for the rock paper scissors game.
Trophazole | Probital | Not much, A bit lower healing and adds a stamina damage component. I made   it a really baby c2. The Metafactor addition is cool and I left that unchanged so eat up!
Rhigoxane | Hercuri | Is geared more towards anti-fire. Heals more when above 50 fire damage,   cools, and lowers fire stacks while in the system. The C2 component is that it can cool to unsafe levels (creative play with cryox/seiver?). Also baby C2.

## Why It's Good For The Game

Rhig and Troph are no brainers considering they're on the same level of ease2make as the C2s and have no downside. They will still be good but they have _some_ C2 component now. Kinda minor compared to the others but I'll see how it plays out.

## Changelog
:cl:
balance: Helbital now is more simple to use and is based off not crit/softcrit/hardcrit. You want to be in hardcrit for the best brute healing.
tweak: Troph has now been converted into a C2 with tweaks, now named Probital. Same recipe. Downside is stamina damage
tweak: Rhig has now been converted into a C2 with tweaks, now named Hercuri. Same Recipe. Downside is it can cool to dangerous levels.
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
Thanks I hate it

**Comment by Tlaltecuhtli** [comment]
tropazole syringe gun new taser

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 1]
```diff
@@ -6,32 +6,34 @@
...
+			M.adjustOxyLoss(2, TRUE)
+		if(SOFT_CRIT) //meh convert
+			thou_shall_heal = round(death_is_coming/47,0.1)
+			M.adjustOxyLoss(1, TRUE)
+		else //no convert
+			thou_shall_heal = round(death_is_coming/45,0.1)
+			good_kind_of_healing = TRUE
+	M.adjustBruteLoss(thou_shall_heal, FALSE)
+
+	if(good_kind_of_healing && !reaping && prob(0.0001)) //janken with the grim reaper!
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: I still think that you should up this prob(0.0001) to something like, I dunno, prob(0.1) (or even prob(1)) or something, because nobody is ever, EVER gonna see this feature otherwise, especially since helbital isn't in Medbay at roundstart and has to be specifically made by a chemist for anyone to even have a CHANCE to get to do the grim reaper bet thing (and then they have to be in hardcrit and roll a chance (which they can only roll once every 2 seconds) that's about as rare as winning a pulse rifle from an arcade machine, then actually accept the bet, even though they're probably being healed by a doctor if they still have helbital in their system while in hardcrit).

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 2]
```diff
@@ -89,6 +91,47 @@
...
+/datum/reagent/medicine/C2/probital
+	name = "Probital"
+	description = "Originally developed as a prototype-gym supliment for those looking for quick workout turnover, this oral medication quickly repairs broken muscle tissue but causes lactic acid buildup, tiring the patient. Overdosing can cause extreme drowsiness. An Influx of nutrients promotes the muscle repair even further."
+	reagent_state = SOLID
+	color = "#FFFF6B"
+	overdose_threshold = 20
+
+/datum/reagent/medicine/C2/probital/on_mob_life(mob/living/carbon/M)
+	M.adjustBruteLoss(-2.25*REM, FALSE)
+	var/ooo_youaregettingsleepy = 3.5
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: I don't like this variable name

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 3]
```diff
@@ -125,6 +168,41 @@
 	..()
 	return TRUE
 
+/datum/reagent/medicine/C2/hercuri
+	name = "Hercuri"
+	description = "Not to be confused with element Mercury, this medicine excels in reverting effects of dangerous high-temperature environments. Prolonged exposure can cause hypothermia."
+	reagent_state = LIQUID
+	color = "#F7FFA5"
+	overdose_threshold = 25
+	reagent_weight = 0.6
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: reagent weight?

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 4]
```diff
@@ -125,6 +168,41 @@
...
+
+/datum/reagent/medicine/C2/hercuri/on_mob_life(mob/living/carbon/M)
+	if(M.getFireLoss() > 50)
+		M.adjustFireLoss(-2*REM, FALSE)
+	else
+		M.adjustFireLoss(-1.25*REM, FALSE)
+	M.adjust_bodytemperature(rand(-25,-5)*(TEMPERATURE_DAMAGE_COEFFICIENT*REM), 50)
+	M.reagents?.chem_temp +=(-10*REM)
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: can this be used to bring chems below 0K?

**Comment by ATH1909** [comment]
> tropazole syringe gun new taser

It'll take 41 ticks for trophazole alone to staminacrit you; just use chloral hydrate or morphine or something instead.

**Comment by ATH1909** [comment]
The "safe" dosage of trophazole should now be 8u, I think. Note that trophazole pills will still have 15u of trophazole in them, so if you eat one, prepare to be staminacritted for about 10 years.

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 5]
```diff
@@ -89,6 +91,47 @@
...
+	..()
+	. = TRUE
+
+/datum/reagent/medicine/C2/probital/overdose_process(mob/living/M)
+	M.adjustStaminaLoss(3*REM, 0)
+	if(M.getStaminaLoss() >= 80)
+		M.drowsyness++
+	if(M.getStaminaLoss() >= 100)
+		to_chat(M,"<span class='warning'>You feel more tired than you usually do, perhaps if you rest your eyes for a bit...</span>")
+		M.adjustStaminaLoss(-100, TRUE)
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: I can't wait for true gamers to intentionally OD themselves on both probital and modafinil, then brush off disabler shots like they're nothing (does the stun from staminacrit get cleared instantly when you leave it, or do you still remain stunned for a while?).

The slowdown from the stamina damage will still be an issue, of course, especially since ignore_slowdown() STILL DOESN'T WORK, but I'm sure someone will find a way around that.

**Comment by ATH1909** [comment]
Also, you might want to proofread that changelog, or at least reformat it so that it's made up of complete sentences.

**Comment by Krysonism** [comment]
Horrendous names. There is some point we got to realise that these "smart" names provide no actual useful info to the players and just create more needless ill will.

Knowing that multiver heals toxins won't allow a total noob to use it safely and effectively and the idea that it will help someone who refuses to read the wiki or ask their fellow doctor is not worth sacrificing all sense of aesthetics over.

**Comment by ATH1909** [comment]
> Horrendous names. There is some point we got to realise that these "smart" names provide no actual useful info to the players and just create more needless ill will.
> 
> Knowing that multiver heals toxins won't allow a total noob to use it safely and effectively and the idea that it will help someone who refuses to read the wiki or ask their fellow doctor is not worth sacrificing all sense of aesthetics over.

Thing is, we've kept this naming style for all of the other cobbychems, so if we left trophazole as trophazole, it'd be an exception to the naming scheme and make things even MORE confusing.

The solution, of course, is to give new chems names that make intuitive sense instead of using any of the prefixes and suffixes of the Cobbychem Naming Scheme (TM). Then again, what each trek chem did (back when we had them) probably wasn't intuitive to newer players...

**Comment by Krysonism** [comment]
> Thing is, we've kept this naming style for all of the other cobbychems, so if we left trophazole as trophazole, it'd be an exception to the naming scheme and make things even MORE confusing.

We should not dig ourself deeper, the solution might be to gradually replace the cobbynames with sensible ones.

It is not unreasonable to require doctors to learn what the chems does from the description, wiki or asking more experienced doctors.

If anything the new names are more confusing since the names are a lot more similar: musiver and multiver for example.

**Comment by ExcessiveUseOfCobblestone** [comment]
If you want to redo the names so long as there's a system that categorizes damage types and tries to avoid realistic names (so that we're not bound to realistic attributes and can balance items freely) be my guest, otherwise please keep your comments about the overarching naming scheme to the appropriate area which is not this PR unless these names are "bad" within that framework.

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm (hunk 5 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: modafinil has it's own issues, if you want to get popped with lethals on top of that go ahead 😏

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm (hunk 4 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: chem temp is effected by bodytemp so I haven't been able to get it below in practical testing, but It's theoretically possible.

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm (hunk 2 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: oooooOOoOOoo you want to resolve this review

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: no

**Comment by Bawhoppen** [comment]
everything should just be named as a variant of cordrazine... it's the only way to be consistent with the lore
eg: helbital -> dicordrazine
trophazole ->tetracordrazine
rhigoxane ->ditribicordazine

**Comment by ExcessiveUseOfCobblestone** [comment]
benzeneazine

**Comment by dr-bee** [comment]
Could you just bake all the healing from Trophazole into the healing peptides instead of making another medicine that is better used as a poison? Having Trophazole do nothing without eating would be a better downside as you have to risk fat slowdown for healing.

**Comment by wesoda25** [comment]
chem names ingame should be 3 randomly generated letters and then 6 randomly generated (maybe 7 if we are feeling daring) numbers for optimal player confusion

**Comment by Krysonism** [comment]
> 
> 
> Could you just bake all the healing from Trophazole into the healing peptides instead of making another medicine that is better used as a poison? Having Trophazole do nothing without eating would be a better downside as you have to risk fat slowdown for healing.

I think moving all the healing to the peptides and swapping the stamina to stomach damage might be an interesting idea.

**Comment by HomocidlePancakes** [comment]
Why do you name these in such a dumb way?

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 6]
```diff
@@ -213,18 +213,18 @@
...
 	required_reagents = list( /datum/reagent/medicine/mannitol = 2, /datum/reagent/water = 2, /datum/reagent/impedrezene = 1)
 
-/datum/chemical_reaction/rhigoxane
-	name = "Rhigoxane"
-	id = /datum/reagent/medicine/rhigoxane
-	results = list(/datum/reagent/medicine/rhigoxane/ = 5)
+/datum/chemical_reaction/hercuri
+	name = "Hercuri"
+	id = /datum/reagent/medicine/C2/hercuri
+	results = list(/datum/reagent/medicine/C2/hercuri/ = 5)
```
Comment (Fikou) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: should that / be there

**Comment by ATH1909** [comment]
> Could you just bake all the healing from Trophazole into the healing peptides instead of making another medicine that is better used as a poison? Having Trophazole do nothing without eating would be a better downside as you have to risk fat slowdown for healing.

It's, uh, not better used as a poison. There are much better stunning/knockout chems that are easy to make that you can use that WON'T heal the damage that you're trying to do to your victim(s).

File: code/modules/reagents/chemistry/recipes/medicine.dm (hunk 6 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/recipes/medicine.dm]:
```suggestion
	results = list(/datum/reagent/medicine/C2/hercuri = 5)
```

**Comment by FancyCyclops** [comment]
> It's, uh, not better used as a poison. There are much better stunning/knockout chems that are easy to make that you can use that WON'T heal the damage that you're trying to do to your victim(s).

Sanguibital.

**Comment by MarioWizard119** [comment]
These chems have a very high potential for abuse, crew and antag. A deliberate Probital OD combined with modafinil and morphine will allow people to shrug off disablers and stun batons. It could also be used to quickly incapacitate someone via syringe gun with bone hurting juice, haloperidol and tirimol. More murderbone tools! Yay! Hercuri also has a high potential for abuse. A deliberate OD combined with cryoxadone makes an extremely strong cure-all pill, but without any of the limitations that keep cryo in check (I.e. the patient being stuck in a tube/under a shower for their treatment). If you though goonchem was brainless and homogenous, imagine the new chem meta being making a grand total of three meds: hercuri OD, cryoxadone, and pent acid (to purge after you’re through healing, cryox already heals toxin).

What makes a good medical main is not whether he chooses to use libital or trophazole. The smart ones will use libital AND trophazole at the same time. The skill in a medical main is never in mixing or using any one single chem, it’s making and mixing several different ones to get people on their feet (or in their grave) quicker. The more recipes you know and the more chems you can make before the bodies start rolling in, the quicker you can get them back up (or rendered fubar) when shit does hit the fan. If a doctor does nothing but use one or two chems because “fuck organ damage”, that’s on them for not wanting to get better at the game. You can remove cat 3s but that will not fix stupidity. No one single chemical can or should be used for everything (be it because it’s crazy overpowered or the only thing that won’t kill you), and chem’s content and balance shouldn’t be centered around lone chemicals (see the conglomerate clusterfuck that is goon’s secret chems).

But making meds deliberately incompatible is also a bad idea as it significantly lowers the skill ceiling as far as what treatments can be given. People will generally just stick to one med and use that for everything. Just look at the toxin meds, since most of them purge each other, people make either seiver or pentetic acid and be done with it. There isn’t much room for improvement if you’re forced into picking a single chem.

**Comment by AdamElTablawy** [comment]
somehow literally everything you just said is wrong, @MarioWizard119 

like, these aren't poisons. they're not even that close. they're not even really abusable, there's better chems to abuse. they're just decent as described. these chems aren't incompatible with other chems, either. they can still be mixed.

**Comment by trollbreeder** [comment]
Honestly, in my opinion, do not put me on a cross, i think that chemicals are more engaging than surgery.

I'll give you an example. Assistant A eats a floorpill and gets dragged to medbay with a mix of poisons. Once i forcefeed him multiver, i have to turn the stasis bed on and off to make sure that A. his lungs dont snap in half, and B. that the poisons he has doesnt kill him in the meantime, which means injecting a syringe of syriniver once in a while to keep the toxin damage under control. After all that, assistant A is now fine and i am satisfied with my medical gameplay. If things do go wrong, that's some more medical drama and i have to either take lungs from the techfab or i have to check the organ smartfridge or i have to take some from a monkey or i'll have to perform lobectomy and make sure i dont fuck up again. I am engaged even more as i have to make sure the remaining multiver isn't too dangerous for the new set of lungs, and that assistant A doesn't suffocate to death while i try to get the multiver in his bloodstream to metabolize to safe levels. Salbutamol from pharmacy it is. **The most viable option is the most fun option and practically the only option. This is how it should be done.**

Contrary to this next example. Assistant B forgot how to switch hands and has brute damage all over. I just need him on the stasis bed, pull out the surgery equipment, tend wounds spam and AFK until he's fine. That wasn't a challenge, that was 20 seconds of waiting while holding a hemostat. If i had used libital, that would've been far more engaging. But you know that people love to optimize the fun out of the game. **You have to show that whatever is the most fun option is the most viable option.**

TL:DR; Nerf tend wounds way down the shitter, make C2 chems the meta instead.

**Comment by lolman360** [comment]
why the rename
old names good cobbynames bad

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm (hunk 5 above)
Comment (MarioWizard119) [reply code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: @ATH1909 Morphine

**Comment by ATH1909** [comment]
> why the rename
> old names good cobbynames bad

In cobby's defense, the new names help people who haven't paid attention to the changelog realize that these chems have been changed/updated.

**Comment by MarioWizard119** [comment]
@AdamElTablawy Care to elaborate on the words you put in my mouth? I don’t think you read my post clearly, much less the code.

I never said these were poisons, most all of the medicines can kill you if you try hard enough.

As for these chems being abusable, on a Probital OD, it causes you to rapidly take stamina damage, and once you take 100, it removes all of your stamina damage, putting you to sleep for 10 seconds. Combine that with Modafinil, a chem that heals sleep, and Morphine, a chem that negates slowdowns, even from stamina damage, and you have yourself a disabler neutering cocktail. Getting stamina crit with this mix in your system would be about as bad as a short knockdown.

With Hercuri, it cools your body temp a random amount between 7.5K and 37.5K, boosted to between 22.5K and 52.5K on overdose. Using a deliberate overdose, you can very quickly get to healing temperatures of cryoxadone (has to be below 273K, humans have a body temp of 310K (humans also take damage at below 270K, so come to think of it, this “baby c2” would reach damaging levels quickly)) without actually having to use the cryo or a shower. Cryox + Hercuri would become the new cure all pill. There’s a reason rhigoxane also makes you take burn damage as well as the deep freeze on overdose, so you can’t do this (yes you could do this with frost oil, but you’d need ice peppers/slimes to pull this off).

**Comment by adamkad1** [comment]
Pretty sure rhig could do burn damage before. also this is dummmm

**Comment by ExcessiveUseOfCobblestone** [comment]
it was capped at `BODYTEMP_NORMAL` for metab, it made you a bit chillier on apply.
